src/components/DrinksList/DrinksListItem.jsx

```diff
@@ -0,0 +1,36 @@
+import {
+    DrinksItem,
+    DeleteButton,
+    SeeMoreButton,
+    DrinkPhoto,
+    BtnWrapper,
+    DrinkTitle,
+    DrinkDesc,
+    TitleWrapper,
+    DrinkAlc
+} from './DrinksList.styled';
+import { useDispatch } from "react-redux";
+import { deleteDrink } from "../../../redux/drinks/operations";
+import sprite from './svg/sprite.svg';
+export const DrinksListItem = ({id, drink}, apiPath) => {
```

Comment:
Don't understand what this id is stands for, because you didn't use id property in DrinkList.jsx file where you using this component